app/main.py

```diff
@@ -1,13 +1,47 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8",
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {
+        "errors": [
+            {
+                "line": error["line_number"],
+                "column": error["column_number"],
+                "message": error["text"],
+                "name": error["code"],
+                "source": "flake8",
+            }
+            for error in errors
+        ],
+        "path": file_path,
+        "status": "passed" if len(errors) == 0 else "failed"
```

Comment:
In python you can just do "if errors", "if not errors". Python will automatically check if there is at least 1 element in this list. If use -> it'll be True, otherwise False